roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: "StackStorm_{{ st2_pkg_repo }}"
+    description: "StackStorm_{{ st2_pkg_repo }}"
+    file: "StackStorm_{{ st2_pkg_repo }}"
+    baseurl: https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/el/{{ ansible_distribution_major_version }}/$basearch
+    repo_gpgcheck: yes
+    gpgkey: "https://packagecloud.io/StackStorm/{{ st2_pkg_repo }}/gpgkey"
+    sslcacert: /etc/pki/tls/certs/ca-bundle.crt
+    metadata_expire: 300
+    gpgcheck: no
+    enabled: yes
+    sslverify: yes
+
+# Fixes "Failure talking to yum: Cannot retrieve repository metadata (repomd.xml) for repository: StackStorm_stable. Please verify its path and try again" when installing st2
+- name: Update ca-certificates package
+  become: yes
+  package:
+    name: ca-certificates
+    state: latest
```

Comment:
@Mierdin It's not clear how https://github.com/StackStorm/ansible-st2/commit/9c52c7baa597c4645db3f98ed2742f76ac5bb2b8 fixes the issue with outdated `ca-certificates`.

According to http://stackoverflow.com/questions/26734777/yum-error-cannot-retrieve-metalink-for-repository-epel-please-verify-its-path and http://unix.stackexchange.com/questions/21310/yum-cannot-retrieve-repository-centos-6 the 2 possible fixes for outdated `EL6` are: 
- use `http` instead of `https` for package repo
- update `ca-certificates` package

How `yum -y repolist` fixes that? I'm probably don't have enough context here.
Additionally, can you reproduce this issue? As I understood it happens in some outdated `EL 6.5` distro.

I'd say, - we don't support some outdated RHEL6.5 which was released in `2013`.

So after all, - I'm +1 to remove this block completely :) WDYT?